mycore-restapi/src/main/java/org/mycore/restapi/v2/MCRRestObjects.java

```diff
@@ -147,7 +162,34 @@ public Response listObjects() throws IOException {
             .lastModified(lastModified)
             .build();
     }
-
+    @POST
+    @Operation(
+        summary = "Create a new empty MyCoRe Object",
```

Comment:
Why start with an empty object? I would prefer to start with an object that already has metadata but a zero integer id: "mir_mods_00000000"

Reply:
I wanted to keep thing simple and let MyCoRe create the "proper" framing.

Reply:
It is common sense in REST APIs to create new objects using POST on /objects (without id), but it may be necessary to proide some initial metadata (mandatory fields, ...)